typescript/phoenix-sdk/src/client.ts

```diff
@@ -5,6 +5,19 @@ import { Token } from "./token";
 import { Market } from "./market";
 import { Trader } from "./trader";
 
+export enum Cluster {
+  MainnetBeta = "mainnet-beta",
+  Devnet = "devnet",
+  Localhost = "localhost",
+}
+
+export const clusterFromEndpoint = (endpoint: string): Cluster => {
```

Comment:
let's put this stuff in a utils/solana.ts file

nit: use function() for definitions rather than arrow functions for consistency